services/register.js

```diff
@@ -0,0 +1,47 @@
+const AWS = require('aws-sdk');
+AWS.config.update({
+    region: 'us-east-1'
+})
+const utils = require('../utils/utils')
+const bcrypt = require('bcryptjs')
+
+const dynamodb = new AWS.DynamoDB.DocumentClient();
+const userTable = 'jinemister-users';
+
+async function register(userInfo) {
+    const name = userInfo.name;
```

Comment:
Update based on schema, also in the issue comment that we are retaining name since it is useful as well.

Reply:
Make sure to update everything.